rfd/0081/README.md

```diff
@@ -4,3 +4,58 @@ state: predraft
 ---
 
 # RFD 81 servicesHealth implementation
+
+The Mariposa (RFD36) health check agent will be responsible for monitoring the health of user services. The Health Check agent process(es) will read user defined health check definitions from the Project service, and coordinate which health check process will be responsible for running any given definition. The Project service is read at agent start time, then polled on an interval. There's an endpoint exposed by the agent that will trigger a poll as well.
```

Comment:
...and now you've caught me eliding a separate problem into this: the sdc-nat zone is not HA. So, I'm not suggesting that we spawn the agent in the zone as much as I was suggesting spawning the zone. 

However, when called out explicitly, it feels like I might be mixing concerns to have the convergence service spawn the nat zone. Or, it could just be that we're persisting the nat zone name after its become something else.